.github/workflows/generator_container_slsa3.yml

```diff
@@ -42,6 +42,9 @@ on:
         description: "Username when publishing to provenance repository (option 'provenance-repository') instead of image repository."
       provenance-registry-password:
         description: "Password when publishing to provenance repository instead of image repository."
+      token:
```

Comment:
the token is available without the need to pass it explicitly, so I don't think we need this option?

Reply:
it will be a different personal access token